deploy/server-setup-guide.md

```diff
@@ -0,0 +1,118 @@
+# Server setup guide
+
+System and network requirements:
+- Ubuntu 18.04 LTS
+- Public static IP address
+
+### Prepeare system:
+#### 1. Upgrade system packages
+```sh
+sudo apt update
+sudo apt upgrade
+```
+#### 2. Install software dependencies
+```sh
+sudo apt install git python3 python3-pip nginx
+```
+#### 3. Install NodeJS v14
+```sh
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
+sudo apt install nodejs
+```
+
+### Set up gunicorn server
+#### 1. Clone the codebase
+```sh
+cd ~
+git clone https://github.com/WebUraXalys/blackout.git
+```
+#### 2. Select the desired branch
+```sh
+git switch <DESIRED BRANCH>
+```
+#### 3. Set up the virtual environment
+```
+python3 -m venv venv
+./venv/bin/activate
+```
+#### 4. Install Python requirements
+```sh
+pip3 install -r ~/blackout/blackout/requirements.txt
```

Comment:
Return the following error:
`Command "/home/ubuntu/venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-voylz75b/gevent/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-dovv00b2-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/ubuntu/venv/include/site/python3.6/gevent" failed with error code 1 in /tmp/pip-build-voylz75b/gevent/`